lib/puppeteer-to-v8.js

```diff
@@ -1,7 +1,24 @@
 class PuppeteerToV8 {
-  // constructor () {}
-}
+  constructor (coverageInfo) {
+    this.coverageInfo = coverageInfo
+  }
+
+  setCoverageInfo (coverageInfo) {
+    this.coverageInfo = coverageInfo
+  }
 
-module.exports = () => {
-  return new PuppeteerToV8()
+  convertCoverage () {
+    // Iterate through coverage info and create IDs
+    let id = 0
+
+    return this.coverageInfo.map(coverageItem => {
+      return {
+        scriptId: id++,
+        url: coverageItem.url,
```

Comment:
once Robert finishes his work,  we'll want to store the final path that will be saved to disk rather than the original URL.

I'm fine with landing it without this at the outset, just noting here that we need to update.